projects/rulenode-core-config/src/lib/components/action/azure-iot-hub-config.component.html

```diff
@@ -0,0 +1,102 @@
+    <mat-error *ngIf="azureIotHubConfigForm.get('clientId').hasError('required')">
+      {{ 'tb.rulenode.device-id-required' | translate }}
+    </mat-error>
+  </mat-form-field>
+  <mat-expansion-panel class="tb-mqtt-credentials-panel-group">
+    <mat-expansion-panel-header>
+      <mat-panel-title translate>tb.rulenode.credentials</mat-panel-title>
+      <mat-panel-description>
+        {{ azureIotHubCredentialsTypeTranslationsMap.get(azureIotHubConfigForm.get('credentials').get('type').value) | translate }}
+      </mat-panel-description>
+    </mat-expansion-panel-header>
+    <section formGroupName="credentials" fxLayout="column">
+      <mat-form-field class="mat-block">
+        <mat-label translate>tb.rulenode.credentials-type</mat-label>
+        <mat-select formControlName="type" required>
+          <mat-option *ngFor="let credentialsType of allAzureIotHubCredentialsTypes" [value]="credentialsType">
+            {{ azureIotHubCredentialsTypeTranslationsMap.get(credentialsType) | translate }}
+          </mat-option>
+        </mat-select>
+        <mat-error *ngIf="azureIotHubConfigForm.get('credentials').get('type').hasError('required')">
+          {{ 'tb.rulenode.credentials-type-required' | translate }}
+        </mat-error>
+      </mat-form-field>
+      <section fxLayout="column" [ngSwitch]="azureIotHubConfigForm.get('credentials').get('type').value">
+        <ng-template ngSwitchCase="anonymous">
+        </ng-template>
+        <ng-template ngSwitchCase="sas">
+          <mat-form-field class="mat-block">
+            <mat-label translate>tb.rulenode.sas-key</mat-label>
+            <input type="password" required matInput formControlName="sasKey">
+            <mat-error *ngIf="azureIotHubConfigForm.get('credentials').get('sasKey').hasError('required')">
+              {{ 'tb.rulenode.sas-key-required' | translate }}
+            </mat-error>
+          </mat-form-field>
+          <tb-file-input formControlName="caCert"
+                         inputId="caCertSelect"
+                         [existingFileName]="azureIotHubConfigForm.get('credentials').get('caCertFileName').value"
+                         (fileNameChanged)="azureIotHubConfigForm.get('credentials').get('caCertFileName').setValue($event)"
+                         requiredAsError
+                         label="{{'tb.rulenode.azure-ca-cert' | translate}}"
+                         noFileText="tb.rulenode.no-file"
+                         dropLabel="{{'tb.rulenode.drop-file' | translate}}">
+          </tb-file-input>
+        </ng-template>
+        <ng-template ngSwitchCase="cert.PEM">
+          <tb-file-input formControlName="caCert"
+                         inputId="caCertSelect"
+                         [existingFileName]="azureIotHubConfigForm.get('credentials').get('caCertFileName').value"
+                         (fileNameChanged)="azureIotHubConfigForm.get('credentials').get('caCertFileName').setValue($event)"
```

Comment:
Need change `get('credentials').get('caCertFileName')` to `get(‘credentials.caCertFileName')`